app/uk/gov/hmrc/taxhistory/services/EmploymentHistoryService.scala

```diff
@@ -44,7 +44,7 @@ class EmploymentHistoryService @Inject()(
 
   def getEmployments(nino: Nino, taxYear: TaxYear)(implicit headerCarrier: HeaderCarrier): Future[List[Employment]] = {
     getFromCache(nino, taxYear).flatMap { paye =>
-      logger.warn("Returning result from getEmployments")
+      logger.info("Returning result from getEmployments")
```

Comment:
I think this (and other similar ones below) should be debug level based on the guidance.

Reply:
:+1: